torch/csrc/autograd/variable.cpp

```diff
@@ -226,12 +152,6 @@ const std::shared_ptr<Function>& Variable::grad_fn() const {
   }
 }
 
-void Variable::DifferentiableViewImpl::release_resources() {
-  auto diff_view_meta = static_cast<Variable::DifferentiableViewMeta*>(get_autograd_meta());
-  diff_view_meta->base_.reset();
```

Comment:
where does this `reset` live now?

Reply:
I just added it in https://github.com/pytorch/pytorch/pull/17072/files#diff-77a6f3462f2233b921d3042412fed6d3R126. The new call path is `TensorImpl::release_resources()` -> `autograd_meta_.reset()` -> `DifferentiableViewMeta::~DifferentiableViewMeta()`  -> `base_.reset()`, which is equivalent to the old code path.